rotkehlchen/tests/api/test_custom_assets.py

```diff
@@ -824,3 +829,91 @@ def assert_db() -> None:
         contained_in_msg='Tried to initialize an asset out of a non-string identifier',
         status_code=HTTPStatus.BAD_REQUEST,
     )
+
+
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+@pytest.mark.parametrize('start_with_logged_in_user', [True])
+def test_exporting_custom_assets_list(rotkehlchen_api_server, globaldb):
+    """Test that the endpoint for exporting custom assets works correctly"""
+    eth_address = make_ethereum_address()
+    identifier = f'_ceth_{eth_address}'
+    globaldb.add_asset(
+        asset_id=identifier,
+        asset_type=AssetType.ETHEREUM_TOKEN,
+        data=EthereumToken.initialize(
+            address=eth_address,
+            decimals=18,
+            name='yabirtoken',
+            symbol='YAB',
+            coingecko='YAB',
+            cryptocompare='YAB',
+        ),
+    )
+    cursor = globaldb._conn.cursor()
+    cursor.execute('INSERT INTO user_owned_assets VALUES(?)', (identifier,))
+    response = requests.get(
+        api_url_for(
+            rotkehlchen_api_server,
+            'assetsexportingresource',
+        ),
+    )
+    result = assert_proper_response_with_result(response)
+    zip_file = ZipFile(result['file'])
+    data = json.loads(zip_file.read('assets.json'))
+    assert int(data['version']) == GLOBAL_DB_VERSION
+    assert len(data['assets']) == 1
+    assert data['assets'][0] == {
+        'identifier': identifier,
+        'name': 'yabirtoken',
+        'decimals': 18,
+        'symbol': 'YAB',
+        'asset_type': 'C',
+        'started': None,
+        'forked': None,
+        'swapped_for': None,
+        'cryptocompare': 'YAB',
+        'coingecko': 'YAB',
+        'protocol': None,
+        'underlying_tokens': None,
+        'ethereum_address': eth_address,
+    }
+    assets_file = Path(result['file'])
+    assets_directory = assets_file.parent
+    assets_file.unlink()
+    assets_directory.rmdir()
+
+
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+@pytest.mark.parametrize('start_with_logged_in_user', [True])
+def test_importing_custom_assets_list(rotkehlchen_api_server):
+    """Test that the endpoint for importing custom assets works correctly"""
+
+    dir_path = Path(__file__).resolve().parent.parent
+    filepath = dir_path / 'data' / 'exported_assets.json'
+
+    response = requests.put(
+        api_url_for(
+            rotkehlchen_api_server,
+            'assetsexportingresource',
+        ), json={'file': str(filepath)},
+    )
+
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    errors = rotki.msg_aggregator.consume_errors()
+    assert len(errors) == 0
+
+    assert_proper_response_with_result(response)
+    stinch = EthereumToken('0xA0446D8804611944F1B527eCD37d7dcbE442caba')
+    assert stinch.symbol == 'st1INCH'
+    assert len(stinch.underlying_tokens) == 1
+    assert stinch.decimals == 18
+
+    response = requests.post(
```

Comment:
This post here is not useful since it has already been imported. You need to make this a parameters of the test so that it runs both with put and post.